sunspec/core/util.py

```diff
@@ -95,6 +96,12 @@ def data_to_double(data):
         return d[0]
 
 def data_to_str(data):
+
+    # Change the data from bytes string to regular string for python 3
+    # compatibility
+    if sys.version_info > (3,):
+        data = str(data, 'latin-1')
```

Comment:
My comment about UTF-8 wasn't about what you should put in the code but about what might belong in the spec.  Only if it makes it into the spec should it be in the code.  If we take the present state of the SunSpec standard this library should not be encoding or decoding because no coding is specified.  Without a specified coding how would this library know what is correct?